### DISCUSSION TITLE
Orientation

### DISCUSSION THREAD

**Discussion by Jeyadhitya** [github/india#270] [Students]
I wasnt able to attend the future ready talent orientation.
Could some one brief with respect to what happened in orientation?

**Comment by tripti6ty** [comment]
the recorded orientation video is posted on the FRT website. you can check it out.

**Reply by Jeyadhitya** [reply]
Thnx !

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
